.github/workflows/pr.yml

```diff
@@ -527,8 +527,17 @@ jobs:
       JOB_NAME: "functional_virtualbox_macos"
       GOPOGH_RESULT: ""
       SHELL: "/bin/bash" # To prevent https://github.com/kubernetes/minikube/issues/6643
-    runs-on: macos-12
+    runs-on: macos-13
     steps:
+      - name: Check architecture
+        shell: bash
+        run: |
+          if [ $(uname -m) != "x86_64" ] 
+          then 
+              echo "$(uname -m) detected. This test must be run on x86_64"
+              exit -1
+          fi
+          exit 0
```

Comment:
if this "Exit 0" will it still be triggered by the Re-Run job

Reply:
I also suggest that you create an issue in in minikube and also in Github Action itself, so they can fix this for everyone, they should not give users Arm Machines